bin/stack_functions

```diff
@@ -396,3 +396,10 @@ function validate_non_zero_varz {
         echo "$varz = 0" | tee -a $results
     fi;
 }
+
+function varz_value {
+    varz=$1
+    results=$2
+    value=$( printf "%.0f" $(wget 0.0.0.0:8302 -O- | grep $varz | grep -v 'HELP\|TYPE' | awk '{print $2'}) )
+    echo "$varz $value" | tee -a $results
```

Comment:
Follow pattern of how it's done in other places in this file

Reply:
Are you referring to the quotes around the echo string? That has been addressed